client/index.html

```diff
@@ -43,22 +43,21 @@
 		</footer>
 		<div id="main">
 			<div id="windows">
-				<div id="chat">
-					<div class="window" style="display:block">
-						<div class="container">
-							<div class="row">
-								<div class="col-xs-12">
-									<h2 class="title">The Lounge is loading…</h2>
-								</div>
-								<div class="col-xs-12">
-									<p id="loading-page-message">Loading the app… <a href="http://enable-javascript.com/" target="_blank">Make sure to have JavaScript enabled.</a></p>
-									<p id="loading-slow" style="display:none">This is taking longer than it should, there might be connectivity issues.</p>
-									<script async src="js/loading-slow-alert.js"></script>
-								</div>
+				<div id="loading" class="window active">
+					<div class="container">
+						<div class="row">
+							<div class="col-xs-12">
+								<h1 class="title">The Lounge is loading…</h1>
+							</div>
+							<div class="col-xs-12">
+								<p id="loading-page-message">Loading the app… <a href="http://enable-javascript.com/" target="_blank">Make sure to have JavaScript enabled.</a></p>
+								<p id="loading-slow">This is taking longer than it should, there might be connectivity issues.</p>
+								<script async src="js/loading-slow-alert.js"></script>
 							</div>
 						</div>
 					</div>
 				</div>
+				<div id="chat"></div>
```

Comment:
Indeed. I actually started by leaving it in there, but I was forced to cancel everything that the `#chat` was setting, and it was a bit messy to hide the input field.
Making it its own window made more sense and cleaned up things a bit.

I hear you though, so I am now detaching the loading page in the `init` event handler. Doing it earlier (when the sign in page appears) means I have to duplicate for public mode, so it's good like this I believe.